app/uk/gov/hmrc/hec/models/YesNoAnswer.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hec.models
+
+import ai.x.play.json.Jsonx
+import ai.x.play.json.SingletonEncoder.simpleName
+import ai.x.play.json.implicits.formatSingleton
+import cats.Eq
+import play.api.libs.json.Format
+
+sealed trait YesNoAnswer extends Product with Serializable
```

Comment:
it'd be nice to use  `YesNoAnswer` for the `saIncomeDeclared` field for individuals as well and get rid of `SAIncomeDeclared`

Reply:
I don't think it'd break anything since they serialise and deserialise in the same way